### PR TITLE
Nibbler implementation with SSE

### PR DESCRIPTION
**Pull request by Ciroye**

### CODE REVIEW

File: backend/app/api/endpoints/base/context.py [hunk 1]
```diff
@@ -53,11 +53,16 @@ async def get_generative_contexts(model: GetGenerativeContextRequest):
 @router.post("/stream")
 async def stream_images(model_info: GetGenerativeContextRequest):
     async def event_generator():
-        for _ in range(2):
-            data = ContextService().get_generative_contexts(
+        iterations = 0
+        for _ in range(3):
```
Comment (remg1997) [inline backend/app/api/endpoints/base/context.py:57]: Maybe there's a chance to make this a configuration inside the artifacts of the yaml? That way if we want more or less images it's easier to change

File: backend/app/domain/schemas/base/historical_data.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    user_id: int
+
+
+class GetSaveHistoricalDataRequest(BaseModel):
+    task_id: int
+    user_id: int
+    data: str
+
+
+class GetDeleteHistoricalDataRequest(BaseModel):
```
Comment (remg1997) [inline backend/app/domain/schemas/base/historical_data.py:19]: Why is this one called Get too?

File: backend/app/domain/services/base/task_proposals.py [hunk 3]
```diff
@@ -24,6 +26,14 @@ def validate_no_duplicate_task_code(self, task_code: str):
...
+        user_email = self.user_repository.get_user_email(user_id)[0]
+        self.email_helper.send(
+            contact=user_email,
+            cc_contact="dynabench-site@mlcommons.org",
+            template_name="task_proposal_update.txt",
+            msg_dict={"name": name, "code": task_code, "desc": longdesc},
+            subject=f"Proposal for task {task_code}",
+        )
         self.email_helper.send(
             contact="juan.ciro@factored.ai",
```
Comment (remg1997) [inline backend/app/domain/services/base/task_proposals.py:38]: Is this working now?

File: frontends/web/src/new_front/context/CreateInterface/Context.tsx [hunk 4]
```diff
@@ -23,6 +23,8 @@ export const CreateInterfaceProvider: FC<CreateInterfaceProviderProps> = ({
       });
     } else {
       setMetadataExample((prevModelInputs) => {
+        console.log("input", input);
```
Comment (remg1997) [inline frontends/web/src/new_front/context/CreateInterface/Context.tsx:26]: Any reason to leave this one here?

File: backend/app/domain/services/base/task_proposals.py (hunk 3 above)
Comment (Ciroye) [reply backend/app/domain/services/base/task_proposals.py:38]: Yes, tested this morning

File: backend/app/domain/schemas/base/historical_data.py (hunk 2 above)
Comment (Ciroye) [reply backend/app/domain/schemas/base/historical_data.py:19]: My bad

**Review by remg1997 — APPROVED** [review]

**Pull request merged**